rpc/src/tests/ws.rs

```diff
@@ -105,6 +105,7 @@ mod testing {
 		http_client::assert_security_headers_present(&response.headers, None);
 	}
 
+	#[cfg(not(target_os = "windows"))]
```

Comment:
why is this test ignored? :)

Reply:
I did not identify the root of the issue, the test is failing on a connection timeout.